documents/src/open_space_map_constructor_database.uxf

```diff
@@ -0,0 +1,425 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<diagram program="umlet" version="14.3.0">
+  <zoom_level>8</zoom_level>
+  <element>
+    <id>UMLClass</id>
+    <coordinates>
+      <x>72</x>
+      <y>416</y>
+      <w>112</w>
+      <h>112</h>
+    </coordinates>
+    <panel_attributes>Buildings
+bg=yellow
+--
++pk_building_id
++address
++number_floors
+
+</panel_attributes>
+    <additional_attributes/>
+  </element>
+  <element>
+    <id>Relation</id>
+    <coordinates>
+      <x>176</x>
+      <y>432</y>
+      <w>112</w>
+      <h>40</h>
+    </coordinates>
+    <panel_attributes>lt=-
+m1=1
+m2=1..*
+</panel_attributes>
+    <additional_attributes>10.0;20.0;120.0;20.0</additional_attributes>
+  </element>
+  <element>
+    <id>UMLClass</id>
+    <coordinates>
+      <x>104</x>
+      <y>144</y>
+      <w>112</w>
+      <h>136</h>
+    </coordinates>
+    <panel_attributes>Employees
+bg=yellow
+--
++pk_employee_id
++first_name
++second_name
++patronymic
++email
++number_telephone
++employee_status
++specialization
```

Comment:
Add the “specialization” field description note.

Reply:
Fixed